libs/feature/teaching/src/lib/course/courseOverview/courseOverview.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import { ProgressBar, Tab, Tabs } from "@self-learning/ui/common";
+import Link from "next/link";
+import { EnrollmentWithDetails } from "@self-learning/types";
+import { PlayIcon } from "@heroicons/react/24/solid";
+
+export function CourseOverview({ enrollments }: { enrollments: EnrollmentWithDetails[] | null }) {
+	const [selectedTab, setSelectedTab] = useState(0);
+
+	if (!enrollments) {
+		return <p>No enrollments found</p>;
+	}
+
+	const inProgress = enrollments.filter(
+		enrollment => enrollment.completions.courseCompletion.completionPercentage < 100
+	);
+	const complete = enrollments.filter(
+		enrollment => enrollment.completions.courseCompletion.completionPercentage >= 100
+	);
+
+	return (
+		<div className="py-2 px-4">
+			{selectedTab === 0 && (
+				<TabContent
+					selectedTab={selectedTab}
+					setSelectedTab={setSelectedTab}
+					enrollments={inProgress}
+					notFoundMessage={"Derzeit ist kein Kurs angefangen."}
+				/>
+			)}
+			{selectedTab === 1 && (
```

Comment:
ich hab die anforderung nicht mehr im kopf. aber falls absehbar ist, dass es weitere tabs geben soll, sollte eine mehr generische variante gefunden werden wo tabs definition auf react elemente gemapped werden. 

wenn nein, dann ist gut so.

Reply:
Ich wüsste nicht was da noch kommen könnte, entweder ist ein Kurs in Arbeit oder er ist Fertig.